components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.ts

```diff
@@ -34,8 +35,24 @@ import { Regex } from 'app/helpers/auth/regex';
 @Component({
   selector: 'app-policy-add-members',
   templateUrl: './policy-add-members.component.html',
-  styleUrls: ['./policy-add-members.component.scss']
+  styleUrls: ['./policy-add-members.component.scss'],
+  animations: [
```

Comment:
Animations used for the fade in/out of form elements as they are needed.  I feel that we're ok animating height here because there are minimal elements involved and they are sitting in a modal.  This helps the modal resize more smoothly without having to adjust the modals actual height separately.

Reply:
Would probably be helpful to add some in-code comments as to what this whole animation block is doing.